src/pages/patterns/status-indicator-pattern/index.mdx

```diff
@@ -421,82 +391,119 @@ you to use full status indicators with 16px IBM Plex.
 <Row>
 <Column colLg={8}>
 
-![Example of a contextual shape status indicator](./images/status_indicator_9.png)
+![Example of shape indicator paired with a label](./images/status_indicator_10.png)
 
-<Caption>Example of the status label directly next to the shape</Caption>
+<Caption>Example shows shape indicator paired with a label</Caption>
 
 </Column>
 </Row>
 
-#### Legends
-
-Because shape indicators don’t have the added recognition of an icon, it’s even
-more important that they are paired with a status label. Alternately, the
-designer can provide the user with a legend if a status label isn’t an option.
-
-<Row>
-<Column colLg={8}>
-
-![Example of a legend paired with a shape indicator](./images/status_indicator_10.png)
+#### Alignment
 
-<Caption>
-  Example where the status symbol indicator depends on the legend at the top of
-  the page
-</Caption>
-
-</Column>
-</Row>
-
-<Title>Best practices</Title>
+When stacked vertically, ensure the icons remain left-aligned with the
+accompanying text for easier scanning.
 
 <DoDontRow>
 <DoDont
 type="do"
 caption="Do place shape indicators before labels; they can be placed after other text only if there is no character count variation.">
 
-![Do place shape indicators before labels; they can be placed after other text only if there is no character count variation.](./images/status_indicator_11.png)
+![Do place shape indicators before labels; they can be placed after other text only if there is no character count variation.](./images/status-legend-do.png)
 
 </DoDont>
 
 <DoDont
 type="dont"
 caption="Do not place shape indicators after the labels to avoid pushing them out of alignment.">
 
-![Do not place shape indicators after the labels to avoid pushing them out of alignment](./images/status_indicator_12.png)
+![Do not place shape indicators after the labels to avoid pushing them out of alignment](./images/status-legend-dont.png)
 
 </DoDont>
 </DoDontRow>
 
+#### Best practice
```

Comment:
```suggestion
#### Best practices
```

Reply:
I know there is one thing under this section right now, but we usually keep this heading plural and there could be more best practices to add in the future.